views/_layout.ejs

```diff
@@ -18,6 +18,9 @@
 			type="text/css" />
 		<!--  -->
 		<link href="/style.css" rel="stylesheet" type="text/css" />
+		
+		<!-- Favicon -->
+		<link rel="icon" href="https://replit.com/public/icons/favicon-prompt-192.png" type="image/png" sizes="192x192">
```

Comment:
Currently `views/service/` and `views/middleware/` both use their own _layout.ejs, could you add this change to those files as well

Reply:
Ok

Reply:
Done